cluster-topology.md

```diff
@@ -0,0 +1,130 @@
+# Use Cluster API to update Cluster Topology
```

Comment:
```suggestion
# Use Cluster API to change the Cluster Topology
```
nit: Just have to have clearer separation between changing the topology vs update/upgrade Kubernetes